ckcp/gitops.sh

```diff
@@ -333,41 +262,22 @@ install_triggers_interceptors(){
     KUBECONFIG=$KUBECONFIG_KCP oc create secret generic kcp-kubeconfig -n tekton-pipelines --from-file "$KUBECONFIG_KCP" >/dev/null
   fi
```

Comment:
since we are changing it at other places, we can change it here too (replace if loops)

oc create secret generic kcp-kubeconfig --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | KUBECONFIG=$KUBECONFIG_KCP oc apply -f - &>/dev/null
oc create secret generic kcp-kubeconfig -n tekton-pipelines --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | KUBECONFIG=$KUBECONFIG_KCP oc apply -f - &>/dev/null